packages/TesterApp/scripts/release.js

```diff
@@ -0,0 +1,35 @@
+const { execSync } = require('child_process');
+
+function runCommand(command) {
+  try {
+    execSync(command, { stdio: 'inherit' });
+  } catch (error) {
+    console.error(`Error executing command: ${command}`, error);
+    process.exit(1);
+  }
+}
+
+function buildIOS() {
+  runCommand('pnpm bundle:ios');
+  process.chdir('./ios');
+  runCommand('bundle exec pod install');
+  runCommand('npx react-native run-ios --mode "Release"');
+}
+
+function buildAndroid() {
+  console.log('Building Android...');
```

Comment:
let's clean the build output dir for android here as well (and perhaps also remove the console log)